app/views/admin/editions/locked.html.erb

```diff
@@ -0,0 +1,24 @@
+<% page_title @edition.title, @edition.format_name %>
+<% page_class "edition-show" %>
+<% initialise_script "GOVUK.adminEditionShow" %>
```

Comment:
Looking at the code for this it doesn't seem to be something we'll need: https://github.com/alphagov/whitehall/blob/7b5c5a086b89cb62ffba62b152a0a8dcfc10c8e6/app/assets/javascripts/admin/views/editions/show.js